### PR TITLE
Add Splunk Universal Forwarder playbooks

### PR DESCRIPTION
**Pull request by cfarquhar**
This commit adds a playbook to install and configure the Splunk Universal
Forwarder as specified by the SAAC team.

### CODE REVIEW

File: installation/local/roles/splunk/defaults/main.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+# 7.2.1, but it had errors on some of the config pushed down from SAAC's
+# infrastructure
+
+# SAAC maintains the "latest" package [here](https://raxglobal-my.sharepoint.com/:f:/g/personal/alan_rothwell_rackspace_co_uk/EtGIOIFy7BlFqGvid5LoEcUBhq7GylYQ7rpQmpDzHFPoqg)
+
+splunk_deb_url: https://download.splunk.com/products/universalforwarder/releases/7.1.2/linux/splunkforwarder-7.1.2-a0c72a66db66-linux-2.6-amd64.deb
+splunk_deb_md5: 47115146ca10dbc8f00befef04b33cd7
+
+# This is the deployment server specified by the SAAC team
+splunk_deployment_server_uri: 10.12.180.99:8089
```
Comment (absalon-james) [inline installation/local/roles/splunk/defaults/main.yml]: Can we leave the exact location outside of the repo?

Comment (cfarquhar) [reply installation/local/roles/splunk/defaults/main.yml]: Per our conversation, I think internal (non-customer) IPs and hostnames are okay.

**Comment by cfarquhar** [comment]
Rebased to `develop` and updated to remove a Racker's name exposed in a sharepoint link.

**Comment by absalon-james** [comment]
merge this please

**Pull request merged**